### PR TITLE
Template fix 

### PR DESCRIPTION
**Pull request by JaaiDead**
im not sure but this might be the fix for it?

### CODE REVIEW

**Comment by JaaiDead** [comment]
public static final String MOD_ID = "modid";
public static final Logger LOGGER = LoggerFactory.getLogger(MOD_ID);
Trying to add this

File: scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta [hunk 1]
```diff
@@ -12,8 +12,10 @@ public class <%= it.className %> implements ModInitializer {
 	// This logger is used to write text to the console and the log file.
 	// It is considered best practice to use your mod id as the logger's name.
 	// That way, it's clear which mod wrote info, warnings, and errors.
-<% if (it.slf4j) { %>    public static final Logger LOGGER = LoggerFactory.getLogger("<%= it.modid %>");
-<% } else { %>    public static final Logger LOGGER = LogManager.getLogger("<%= it.modid %>");<% } %>
+<% if (it.slf4j) { %>    public static final String MOD_ID = "<%= it.modid %>";
+<% } else { %>    public static final String MOD_ID = "<%= it.modid %>";<% } %>
```
[the remark this answers is not in the document]
Comment (JaaiDead) [reply scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta:16]: oops forgot to remove it

File: scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta [hunk 2]
```diff
@@ -12,8 +12,9 @@ public class <%= it.className %> implements ModInitializer {
 	// This logger is used to write text to the console and the log file.
 	// It is considered best practice to use your mod id as the logger's name.
 	// That way, it's clear which mod wrote info, warnings, and errors.
-<% if (it.slf4j) { %>    public static final Logger LOGGER = LoggerFactory.getLogger("<%= it.modid %>");
-<% } else { %>    public static final Logger LOGGER = LogManager.getLogger("<%= it.modid %>");<% } %>
+        public static final String MOD_ID = "<%= it.modid %>";
+<% if (it.slf4j) { %> public static final Logger LOGGER = LoggerFactory.getLogger("<%= MOD_ID %>");
+<% } else { %> public static final Logger LOGGER = LogManager.getLogger("<%= MOD_ID %>");<% } %>
```
Comment (Octol1ttle) [inline scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta:17]: Indentation needs to be fixed

**Review by Octol1ttle — CHANGES_REQUESTED** [review]

File: scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta (hunk 2 above)
Comment (modmuss50) [reply scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta:17]: Yes, please double check this, its quite finicky to get this right.

**Comment by JaaiDead** [comment]
image

**Comment by dicedpixels** [comment]
Shouldn't the MOD_ID be passed to `getLogger`?

**Comment by JaaiDead** [comment]
Fixed In Indentation Test i suppose its needs to be `public static final Logger LOGGER = LoggerFactory.getLogger(MOD_ID)` ?

File: scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta (hunk 2 above)
Comment (JaaiDead) [reply scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta]: Can you give me an example to how it should look like ?

File: scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta [hunk 3]
```diff
@@ -12,8 +12,11 @@ public class <%= it.className %> implements ModInitializer {
 	// This logger is used to write text to the console and the log file.
 	// It is considered best practice to use your mod id as the logger's name.
 	// That way, it's clear which mod wrote info, warnings, and errors.
-<% if (it.slf4j) { %>    public static final Logger LOGGER = LoggerFactory.getLogger("<%= it.modid %>");
-<% } else { %>    public static final Logger LOGGER = LogManager.getLogger("<%= it.modid %>");<% } %>
+        public static final String MOD_ID = "<%= it.modid %>";
```
Comment (modmuss50) [inline scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta:15]: There should be a single tab, same as the comments above. I would also expect this line to be above the comments.

**Comment by JaaiDead** [comment]
image

File: scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta [hunk 4]
```diff
@@ -12,8 +12,9 @@ public class <%= it.className %> implements ModInitializer {
 	// This logger is used to write text to the console and the log file.
 	// It is considered best practice to use your mod id as the logger's name.
 	// That way, it's clear which mod wrote info, warnings, and errors.
-<% if (it.slf4j) { %>    public static final Logger LOGGER = LoggerFactory.getLogger("<%= it.modid %>");
-<% } else { %>    public static final Logger LOGGER = LogManager.getLogger("<%= it.modid %>");<% } %>
+    public static final String MOD_ID = "<%= it.modid %>";
```
Comment (Octol1ttle) [inline scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta:15]: I believe what modmuss wanted is something like:
```java
    public static final String MOD_ID = "<%= it.modid %>";
    // This logger is used to write text to the console and the log file.
	// It is considered best practice to use your mod id as the logger's name.
	// That way, it's clear which mod wrote info, warnings, and errors.
	<% if (it.slf4j) { %>    public static final Logger LOGGER = LoggerFactory.getLogger("<%= it.modid %>");
	<% } else { %>    public static final Logger LOGGER = LogManager.getLogger("<%= it.modid %>");<% } %>
```

**Review by Octol1ttle — CHANGES_REQUESTED** [review]

File: scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta (hunk 4 above)
Comment (JaaiDead) [reply scripts/src/lib/template/templates/entrypoint/Entrypoint.java.eta:15]: Oh alr I got a bit confused alr I'll change it now

**Review by modmuss50 — APPROVED** [review]

**Comment by JaaiDead** [comment]
@Octol1ttle For Review:D

**Pull request merged**